Update RabbitMQ docs to core V8 (#5230)
Particular/docs.particular.net: transports/rabbitmq/connection-settings_prefetch-control_rabbit_[7,).partial.md

## Controlling the prefetch count

When consuming messages from the broker, throughput can be improved by having the consumer [prefetch](https://www.rabbitmq.com/consumer-prefetch.html) additional messages.

> By default the prefetch count is calculated by multiplying [maximum concurrency](/nservicebus/operations/tuning.md#tuning-concurrency) by the prefetch multiplier of 3. The default prefetch count calculation can be overridden. It can still take into account the configured concurrency settings that is passed as the input to the calculation function.

Reviewer: ```suggestion
By default the prefetch count is calculated by multiplying [maximum concurrency](/nservicebus/operations/tuning.md#tuning-concurrency) by the prefetch multiplier of 3. The default prefetch count calculation can be overridden with a custom algorithm. The configured concurrency setting is passed as an input to the calculation function.
```
